packages/cli/src/collaboration/collaboration.service.ts

```diff
@@ -89,7 +91,10 @@ export class CollaborationService {
 		if (workflowUserIds.length === 0) {
 			return;
 		}
-		const users = await this.userService.getByIds(this.userService.getManager(), workflowUserIds);
+		const users = await this.userRepository.getByIds(
+			this.userService.getManager(),
```

Comment:
not sure why `UserService.getManager` was added, but this is unnecessary, and should be removed. if `userRepository.getByIds` wants to accept a optional transaction, then we should update it to
`async getByIds(ids: string[], transaction: EntityManager = this.manager)` instead of passing around entity-managers like this.

Reply:
`getByIds` requires a transaction because I'm moving all persistence logic to the persistence layer and that is how this method existed originally. We should refactor to create more generalized reusable methods in the repositories, but for now I'd like to focus on finishing moving all persistence logic to the persistence layer, which is a ton of work as it is 🙈